latest.Dockerfile

```diff
@@ -23,12 +23,12 @@ RUN groupadd "runner" && useradd -g "runner" --shell /bin/bash "runner" \
 COPY --from=install /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=install ./runner /home/runner
 
-# install libicu for Ubuntu 25.04
+# Install libicu for GitHub Actions runner compatibility and apply security updates
 # https://github.com/actions/runner/issues/3150
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     libicu-dev \
```

Comment:
Ubuntu 25.10 (Questing Quokka) includes the **libicu-dev** package which provides the International Components for Unicode (ICU) library. This library is essential for Unicode and locale support in C++ and C applications.
Different Ubuntu releases may use different versions of libicu. If you have a specific software that requires a particular version of libicu not included in 25.10, you might need to find alternative ways to install it.

Reply:
It sounds like you don't need to install this library anymore.